assets/app/scripts/app.js

```diff
@@ -189,12 +189,13 @@ angular
     }
   }])
   .run(function($interval, dateRelativeFilter, durationFilter) {
-    $interval(function() {
+    // Use setInterval instead of $interval because we're directly manipulating the DOM and don't want scope.$apply overhead
+    setInterval(function() {
```

Comment:
Should we still use `$interval` and pass `false` for the `invokeApply` param?

https://docs.angularjs.org/api/ng/service/$interval